apps/schema/src/columnTypes.js

```diff
@@ -7,7 +7,8 @@ export default [
   "DATETIME",
   "REF",
   "REF_ARRAY",
-  //depcrecated "MREF",
+  "ONTOLOGY",
+  "ONTOLOGY_ARRAY",
```

Comment:
are these new emx col type ? sould the docs be updated ?

Reply:
docs have been updated in previous pull request, see https://emx2.dev.molgenis.org/apps/docs/#/use_schema?id=columntype